crates/utils/src/utils/validation.rs

```diff
@@ -350,12 +340,11 @@ pub fn build_url_str_without_scheme(url_str: &str) -> LemmyResult<String> {
 }
 
 #[cfg(test)]
-#[allow(clippy::unwrap_used)]
```

Comment:
Nothing really important below, just removing the unwraps in favor of using `LemmyResult` in the tests.